multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/mangathemesia/MangaThemesiaGenerator.kt

```diff
@@ -120,6 +120,7 @@ class MangaThemesiaGenerator : ThemeSourceGenerator {
         SingleLang("White Cloud Pavilion (New)", "https://www.whitecloudpavilion.com", "en", pkgName = "whitecloudpavilionnew", className = "WhiteCloudPavilion"),
         SingleLang("World Romance Translation", "https://wrt.my.id", "id", overrideVersionCode = 10),
         SingleLang("xCaliBR Scans", "https://xcalibrscans.com", "en", overrideVersionCode = 4),
+        SingleLang("Gremory Mangas", "https://gremorymangas.com/", "es", overrideVersionCode = 1),
```

Comment:
Since this is a new extension, and not a change to an existing extension, you don't really need `overrideVersionCode`
```suggestion
        SingleLang("Gremory Mangas", "https://gremorymangas.com/", "es"),
```